### PR TITLE
Uxprod 2613

### PR DESCRIPTION
**Pull request by EstebanSM**
## Purpose

- [UICAT-106 Search authority records](https://issues.folio.org/projects/UICAT/issues/UICAT-106)

- [UICAT-112 Add option to reset all in the search](https://issues.folio.org/projects/UICAT/issues/UICAT-112)

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/Browse/BrowseResults.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ export class BrowseResults extends React.Component<Props, S> {
                         id="refType1"
                         style={{ fontWeight: 'bold', margin: 0, padding: 0 }}
                       >
-                        {'See: '}
+                        See:
```
Comment (ryandberger) [inline src/components/Browse/BrowseResults.js:229]: This should be added to the translation file `en.json`.

File: src/components/Search/Panel/SearchPanel.js [hunk 2]
```diff
@@ -222,6 +232,37 @@ class SearchPanel extends React.Component<P, {}> {
...
+    this.setState({
+      segment: segmentactive,
+      btnSubmitEnabled: false
+    });
+  };
+
+  handleBtnResetAll = () => {
+    const { segment } = this.state;
+    this.changeSegment(segment);
+    // dispatch(resetFilterSearch());
```
Comment (ryandberger) [inline src/components/Search/Panel/SearchPanel.js:249]: *nitpick* Remove commented code.

File: src/redux/epic/epics.js [hunk 3]
```diff
@@ -55,6 +55,59 @@ export const searchEpic = (action$, store) => action$.ofType(ACTION.SEARCH)
...
+        );
+      }).catch(e => of$(marccatActions.fetchFailure(e)))
+  ));
+
+// Para b�squedas Authority
+export const searchAuthEpic = (action$, store) => action$.ofType(ACTION.SEARCHAUTH)
+  .switchMap((d) => concat$(
+    of$(marccatActions.isfetchingSearchRequest(true, d.moreData, d.isFromCat)),
+    getJSON(buildUrl(store.getState(), ENDPOINT.SEARCH_AUTH_URL, `lang=eng&ml=170&q=${d.queryAuth}&from=${d.from}&to=${d.to}&dpo=1&sortBy=${d.isFromCat === 'Y' ? 54 : Selector.get(store, 'settings', 'sortType')}&sortOrder=0`), getHeaders(store.getState()))
+    // getJSON(buildUrl(store.getState(), ENDPOINT.MERGED_SEARCH_URL, `lang=eng&ml=170&qbib=${d.queryBib}&qauth=${d.queryAuth}&from=${d.from}&to=${d.to}&dpo=1&sortBy=${d.isFromCat === 'Y' ? 54 : Selector.get(store, 'settings', 'sortType')}&sortOrder=0`), getHeaders(store.getState()))
```
Comment (ryandberger) [inline src/redux/epic/epics.js:89]: *nitpick* Remove commented code.

Comment (EstebanSM) [reply src/redux/epic/epics.js]: I have removed the commented lines

File: src/components/Search/Panel/SearchPanel.js (hunk 2 above)
Comment (EstebanSM) [reply src/components/Search/Panel/SearchPanel.js]: I have removed the commented lines

File: src/components/Browse/BrowseResults.js (hunk 1 above)
Comment (EstebanSM) [reply src/components/Browse/BrowseResults.js]: I have included the texts in en.json

**Review by matteopascucci — APPROVED** [review]

File: translations/ui-marccat/en.json [hunk 4]
```diff
@@ -49,6 +49,8 @@
   "search.associated.bibliographic": "Associated bibliographic records",
   "search.history.title": "Recent Searches",
   "browse.results.title": "Access Point",
+  "browse.results.see": "See",
+  "browse.results.seefrom": "Seen From",
```
Comment (ryandberger) [inline translations/ui-marccat/en.json:53]: Update this label to be `seenfrom`. Otherwise, looks good.

Comment (EstebanSM) [reply translations/ui-marccat/en.json:53]: ok, very attentive! done

**Review by ryandberger — APPROVED** [review]

**Pull request merged**